_episodes_rmd/06-tidy-data.Rmd

```diff
@@ -0,0 +1,73 @@
+---
+title: "Tidying & packaging datasets"
+teaching: 30
+exercises: 10
+questions:
+- "Which different forms can one dataset have?"
+- "Which advantages and disadvantages do these forms have?"
+- "Which features make a dataset more or less reusable?"
+- "How can we add datasets to R packages?"
+objectives:
+- "Use `tidyr::gather()` to convert wide data to its long form."
```

Comment:
This objective doesn't relate to item 4 of questions. Maybe add to it saying "and adding to package"?

Reply:
More objectives will be added. I have a task list in my staging area ;-)